src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -208,13 +208,13 @@
     <value>The default value of an uninitialized enumeration, just as other value types, is zero. A nonflags-attributed enumeration should define a member by using the value of zero so that the default value is a valid value of the enumeration. If an enumeration that has the FlagsAttribute attribute applied defines a zero-valued member, its name should be ""None"" to indicate that no values have been set in the enumeration.</value>
   </data>
   <data name="EnumsShouldHaveZeroValueMessageFlagsRename" xml:space="preserve">
```

Comment:
Rename resource name to end with `Message`?

Reply:
This was one of the cases I was explaining in the ticket. Currently when we have a rule where the various descriptors share the same title and description but differ by the message we use the following pattern: <RuleName><Message><DescriptorSpecificSuffix>. It would be easier for the analyzer if we were able to use a different pattern, for example <RuleName><Message_><DescriptorSpecificSuffix>.

Reply:
Sounds good. I think you don't need to make this change as part of this PR. I am fine with even not changing the resource name until we actually have an analyzer that checks name and strictly enforces it.